CITATION.cff

```diff
@@ -0,0 +1,55 @@
+cff-version: 1.2.0
+title: DLA-Future
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Raffaele
+    family-names: Solcà
+    orcid: 'https://orcid.org/0009-0009-9346-4376'
+  - given-names: Alberto
+    family-names: Invernizzi
+    orcid: 'https://orcid.org/0009-0004-4948-0313'
+  - given-names: Auriane
+    family-names: Reverdell
+    orcid: 'https://orcid.org/0000-0002-5531-0458'
+  - given-names: John
+    family-names: Biddiscombe
+    orcid: 'https://orcid.org/0000-0002-6552-2833'
+  - given-names: Mikael
+    family-names: Simberg
+    orcid: 'https://orcid.org/0000-0002-7238-8935'
+  - given-names: Rocco
+    family-names: Meli
+    orcid: 'https://orcid.org/0000-0002-2845-3410'
+  - given-names: Guglielmo
+    family-names: Gagliardi
+  - given-names: Andreas
+    family-names: Fink
+  - given-names: Teodor
+    family-names: Nikolov
+  - given-names: Harmen
+    family-names: Stoppels
+  - given-names: Lara
+    family-names: Querciagrossa
+  - given-names: Alo
+    family-names: Roosing
+  - given-names: Alessandro
+    family-names: Colombo
```

Comment:
@gulivarese, @finkandreas, @teonnik, @haampie, @Sely85, @roosinga, @a-colombo do you have an orcid that you'd like to have added here? I either couldn't find one for you or there were multiple yous. Let me know in a comment if you'd like one added.

Even if you bump into this comment sometime in the future after this has been merged feel free to comment anyway, as we can edit past releases.

Reply:
@msimberg dear Mikael, it's so nice to see this project converging! :) You can use the following ORCID for me: 0000-0002-7950-293X